crates/turborepo-dirs/Cargo.toml

```
@@ -8,6 +8,7 @@ license = "MPL-2.0"
 
 [dependencies]
 dirs-next = "2.0.0"
+thiserror.workspace = true
```

guessing yes, but is the same as below?

```suggestion
thiserror = { workspace = true }
```

---

Yeah they're the same, I think cargo defaults to this version if you don't specify things like `features`. I'm sure there's some configuration I can do somewhere to fix this but I haven't taken the time to deal with it yet  😁